### PR TITLE
Cleanup2

### PR DESCRIPTION
**Pull request by SuperSandro2000**
###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/networking/easyrsa/2.x.nix [hunk 1]
```diff
@@ -22,7 +32,7 @@ stdenv.mkDerivation {
     for prog in $(find "$out/share/easy-rsa" -executable -type f); do
       makeWrapper "$prog" "$out/bin/$(basename $prog)" \
         --set EASY_RSA "$out/share/easy-rsa" \
-        --set OPENSSL "${openssl.bin}/bin/openssl" \
+        --set OPENSSL "${lib.getLib openssl}/bin/openssl" \
```
Comment (thiagokokada) [inline pkgs/tools/networking/easyrsa/2.x.nix:35]: This seems wrong:

```
$ nix-repl
nix-repl> toString (lib.getLib pkgs.openssl)
"/nix/store/4frnjf7jr2cc5k5wfqjapvszanmc8nq9-openssl-1.1.1l"

nix-repl> toString openssl.bin
"/nix/store/10jzrky6qp59qm75zwxxjysyzphywznj-openssl-1.1.1l-bin"

$ ls -lah /nix/store/4frnjf7jr2cc5k5wfqjapvszanmc8nq9-openssl-1.1.1l
total 2.7M
dr-xr-xr-x    4 root root     28 dez 31  1969 .
drwxrwxr-t 5948 root nixbld 2.4M nov 10 17:18 ..
dr-xr-xr-x    3 root root     17 dez 31  1969 etc
dr-xr-xr-x    3 root root    107 dez 31  1969 lib
```

Comment (SuperSandro2000) [reply pkgs/tools/networking/easyrsa/2.x.nix:35]: Thanks, you are right

**Review by thiagokokada — APPROVED** [review]

**Comment by thiagokokada** [comment]
Result of `nixpkgs-review pr 145410` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages built:</summary>
  <ul>
    <li>easyrsa2</li>
    <li>fbterm</li>
    <li>fcitx-configtool</li>
    <li>update-resolv-conf</li>
  </ul>
</details>

**Review by 7c6f434c — APPROVED** [review]

**Comment by SuperSandro2000** [comment]
rebase on master to resolve merge conflict

File: pkgs/os-specific/linux/fbterm/default.nix [hunk 2]
```diff
@@ -51,7 +46,6 @@ stdenv.mkDerivation {
   ];
```
Comment (jtojnar) [inline pkgs/os-specific/linux/fbterm/default.nix:46]: Would be also nice to move patches after sources.

Comment (SuperSandro2000) [reply pkgs/os-specific/linux/fbterm/default.nix:46]: Mostly already done in #155226

**Pull request closed without merging**